lib/Basic/Platform.cpp

```diff
@@ -218,6 +218,8 @@ StringRef swift::getMajorArchitectureName(const llvm::Triple &Triple) {
       return "armv7";
     case llvm::Triple::SubArchType::ARMSubArch_v6:
       return "armv6";
+    case llvm::Triple::SubArchType::ARMSubArch_v5:
```

Comment:
This is similar to https://github.com/apple/swift-driver/blob/0f9f247c8cc3d225dbbc263b1ec082bf2c8675fe/Sources/SwiftDriver/Jobs/GenericUnixToolchain%2BLinkerSupport.swift#L45 and its possible Swift-Driver needs to be patched to find Glibc.

Reply:
Patched 
https://gist.github.com/colemancda/df87bab835f431f6b02ea9c795dffcd7
https://github.com/apple/swift-driver/pull/1027